packages/server/src/db/index.js

```diff
@@ -790,31 +790,29 @@ async function sync(tableName, syncKey, updateCols, newRows) {
 }
 
 /**
- * Determines if a user's tenant is the same as the agency's tenant
+ * Determines if all given agencies belong to the same given tenant
  * @param  int        userId
  * @param  int        tenantId
  * @parm   Array[int] agencyIds
  * @return boolean
  * */
 async function inTenant(userId, tenantId, agencyIds) {
```

Comment:
It appears we're no longer using the `userId` arg and all existing callsites pass `user.tenant_id` for `tenantId` anyway (thus checking `user.tenantId === tenantId` would be sorta useless). Should we just remove the `userId` param then?